docs/contributor/troubleshooting.md

```diff
@@ -0,0 +1,11 @@
+# Troubleshooting
+
+This document contains tips and tricks to common problems with the NATS Manager and will be updated continuously.
+
```

Comment:
consider moving the Troubleshooting section from the Installation doc to this doc.